plugins/woocommerce/src/Blocks/Utils/ProductCollectionUtils.php

```diff
@@ -92,4 +99,171 @@ private static function remove_empty_array_recursive( $array ) {
 		}
 		return $array;
 	}
+
+	/**
+	 * Parse editor's location context from WP Post.
+	 *
+	 * Possible contexts:
+	 * - post
+	 * - page
+	 * - product
+	 * - product-archive
+	 * - cart
+	 * - checkout
+	 * - catalog
+	 * - order
+	 *
+	 * @param WP_Post $post The Post instance.
+	 *
+	 * @return string Returns the context.
+	 */
+	public static function parse_editor_location_context( $post ) {
+		$context = 'site';
+
+		if ( ! $post instanceof \WP_Post ) {
+			return $context;
+		}
+
+		$post_type = $post->post_type;
+		if ( ! in_array( $post_type, array( 'post', 'page', 'wp_template', 'wp_template_part' ), true ) ) {
+			return $context;
+		}
+
+		if ( in_array( $post_type, array( 'wp_template', 'wp_template_part' ), true ) ) {
+
+			$name = $post->post_name;
+			if ( false !== strpos( $name, SingleProductTemplate::SLUG ) ) {
+				$context = 'product';
```

Comment:
In the editor I used `single-product`. Worth unifying.

```suggestion
				$context = 'single-product';
```

I don't have strong opinion and if you feel order is better, please update in [here](https://github.com/woocommerce/woocommerce/blob/fd85199f9a27202ab25c6ca536a17189d2da8187/plugins/woocommerce-blocks/assets/js/blocks/product-collection/tracks-utils.ts#L8)

Reply:
Done in https://github.com/woocommerce/woocommerce/pull/46466/commits/da2f7b41e564489fb2c59b1700b831da7f1117e2

I don't have a strong opinion either. Both work fine, so I'm keeping the 'single-product' for consistency.